xonsh/tools.py

```diff
@@ -629,14 +629,23 @@ def swap(namespace, name, value, default=NotImplemented):
         setattr(namespace, name, old)
 
 #
-# Validators and contervers
+# Validators and converters
 #
 
 
 def is_int(x):
     """Tests if something is an integer"""
     return isinstance(x, int)
 
+def is_str_as_int(x):
+    """Tests if something is an integer"""
+    if isinstance(x, str):
+        try:
+            int(x)
+        except ValueError:
+            return False
+    return True
```

Comment:
Also I am not sure that this is the correct return value, if I pass in a list, this will return True.